src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -497,39 +339,101 @@ Below is a list of the available parameters for *touch-action*.
   </thead>
   <tbody>
     <tr>
-      <td data-th="Property"><code>touch-action: auto</code></td>
-      <td data-th="Description">
-        The browser will add the normal touch interactions which it supports. For example, scrolling in the x-axis, scrolling in the y-axis, pinch zoom and double tap.
-      </td>
-    </tr>
-    <tr>
       <td data-th="Property"><code>touch-action: none</code></td>
       <td data-th="Description">No touch interactions will be handled by the browser.</td>
     </tr>
     <tr>
-      <td data-th="Property"><code>touch-action: pan-x</code></td>
-      <td data-th="Description">Only horizontal scrolling will be handled by the browser; vertical scrolling and gestures will be disabled.</td>
+      <td data-th="Property"><code>touch-action: pinch-zoom</code></td>
+      <td data-th="Description">Disables all browser interactions like
+      `touch-action: none` apart from pinch-zoom, which is still handled by the
```

Comment:
This is the only reference for the pinch-zoom property,  the others are referring to the action of pinch zooming.